src/Forms/CartForm.jsx

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+
+import Button from "../components/Button/Button.jsx";
+import Icon from "../components/Icon/Icon.jsx";
+import Group from "../components/Group/Group.jsx";
+import Link from "../components/Link/Link.jsx";
+import Section from "../components/Section/Section.jsx";
+import Text from "../components/Text/Text.jsx";
+import TextInput from "../components/TextInput/TextInput.jsx";
+import Select from "../components/Select/Select.jsx";
+
+class CartForm extends React.Component {
+  static CURRENCIES = [
+    { value: "gbp", label: "GBP" },
+    { value: "usd", label: "USD" }
+  ];
```

Comment:
I'm not entirely certain what to do with this – should it be driven by the account's supported currencies from the example backend, or just a static list like this?

Reply:
I think a static list is fine for now!